packages/analytics/src/index.tsx

```
@@ -2,23 +2,23 @@ import * as React from 'react';
 
 	React.useEffect(() => {
-		txPromise.then(() => setIsReady(true));
+		i18nPromise.then(() => setIsReady(true));
 	}, []);
 
 	return (
 		<>
 			<Notice isDismissible={false} className="woocommerce-pos-upgrade-notice">
-				{t('Do you want analytics for your POS orders?', { _tags: 'wp-admin-analytics' })}{' '}
+				{t('Do you want analytics for your POS orders?')}{' '}
 				<a target="_blank" href="https://wcpos.com/pro">
```

Biome (2.3.13)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)